app/models/tslr_claim.rb

```diff
@@ -46,6 +52,8 @@ class TslrClaim < ApplicationRecord
 
   validates :mostly_teaching_eligible_subjects, on: [:"mostly-teaching-eligible-subjects", :submit], inclusion: {in: [true, false], message: "Select either Yes or No"}
 
+  validates :payroll_gender,                    on: [:gender, :submit], presence: {message: "Choose the option for the gender your school’s payroll system associates with you"}
```

Comment:
Change the context to `:"payroll-gender"` if renaming the page.

Reply:
Reckon we keep the page slug for now.